doc/api/globals.md

```diff
@@ -592,6 +592,10 @@ added: v22.4.0
 
 A browser-compatible implementation of [`localStorage`][]. Data is stored
 unencrypted in the file specified by the [`--localstorage-file`][] CLI flag.
+Data is stored in memory, with a storage quota of 10 MB.
+`localStorage` data persists within the current browser process and across
+page refreshes for the same origin,but it's not shared across different
+browser tabs, windows, or server-side rendering (SSR) contexts.
```

Comment:
What was "current browser" mean? The storage isn't in the browser, but in Node.js, unless I'm missing something

Reply:
Need to be `current browser's process`. I'll update it.

Reply:
No, I'm not understanding why a browser is even mentioned. I might be missing something, but isn't this done completely independent of the browser?

Reply:
I get it now. `current browser` is not true on that. It shouldn't be there.

Reply:
While browser is removed, there aren't any 'tabs', 'windows', or 'contexts' in Node.js. (Along with other things mentioned). Could you reword this to fit Node.js instead of the Browser? Thanks ! :-)